content/webapp/pages/exhibition-guide.tsx

```diff
@@ -4,20 +4,26 @@ import { fetchExhibitionGuide } from '../services/prismic/fetch/exhibition-guide
 // import { transformQuery } from '../services/prismic/transformers/paginated-results';
 import { transformExhibitionGuide } from '../services/prismic/transformers/exhibition-guides';
 import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
-import { FC } from 'react';
-import { GetServerSideProps } from 'next';
-import { /* appError, */ AppErrorProps } from '@weco/common/views/pages/_app'; // TODO
+import React, { FC } from 'react';
```

Comment:
We shouldn't need to import `React` directly – there's some magic transpilation that means it's always available for us in component files (a part of NextJS I _think_, but perhaps we added it ourselves. I'll find out).

Reply:
I think webstorm was trying to be helpful here, so thank you for flagging that